misc/server_log.yml

```diff
@@ -37,6 +37,11 @@ loggers:
     - core_tornado_debug_log_handler
     level: DEBUG
     propagate: true
+  sqlalchemy.engine:
+    handlers:
+      - server_handler
+    level: INFO
+    propagate: true
```

Comment:
this will cause duplicate logs, as the `server_handler`  handler is both attached at the root and here.